caffe2_builders/v140/msbuild.bat

```diff
@@ -0,0 +1,29 @@
+if not exist %CAFFE2_ROOT%\build\%CONFIG% (
+    mkdir %CAFFE2_ROOT%\build\%CONFIG%
+)   
+cd %CAFFE2_ROOT%\build\%CONFIG%
+
+if %CMAKE_GENERATOR% EQU "Visual Studio 14 2015 Win64" (
+
+    cmake %CAFFE2_ROOT% -G%CMAKE_GENERATOR% ^
+                             -DUSE_OPENCV=ON ^
+                             -DCMAKE_BUILD_TYPE=%CONFIG% ^
+                             -DCMAKE_INSTALL_PREFIX=%CAFFE2_ROOT%\build\%CONFIG%\install ^
+                             -DCMAKE_CXX_COMPILER=%CXX% ^
+                             -DCMAKE_C_COMPILER=%CC% ^
+                             -DCMAKE_LINKER=%CMAKE_LINER% ^
+                             -DINCLUDE_EXPERIMENTAL_C10_OPS=OFF ^
+                             -DBUILD_SHARED_LIBS=ON ^
+                             -DBUILD_BINARY=ON
+                                 
```

Comment:
We'll have to catch errors here, otherwise, the script will complete with exit code 0 even if there are actually errors. Consider using something like `if errorlevel 1 exit /b 1`